VATSpy.dat

```diff
@@ -18304,9 +18304,10 @@ LPPO|Santa Maria|LPAZ|LPPO
 LPPO-TMA|Santa Maria Radar|LPPO|LPPO-TMA
 LPAM|Lisboa Military||LPPC
 LQSB|Sarajevo|LQSA|LQSB
-LRBB|Bucharest||LRBB
-LRBB-A|Bucharest|LRBB_A|LRBB-A
-LRBB-N|Bucharest|LRBB_N|LRBB-N
+LRBB|Bucuresti ACC (KONEL) - Bucharest||LRBB

```

Comment:
Having sectornames (KONEL) attached to the main FIR is not something we usually do, you sure you want KONEL to be displayed on your main FIR?

Reply:
Hello,
KONEL sector is pretty much known in Bucharest FIR. I named all subsectors with their combined name used IRL, this one would remain nameless, which, when they are all online, would look different.
And the Bandbox sector doesn't even exist IRL theoretically, so I don't know what I could name it. Any ideas?